docs/versioning/olm.md

```diff
@@ -0,0 +1,98 @@
+## How to create a MGC OLM bundle, catalog and how to install MGC via OLM
+
+### Creating the bundle
+    :exclamation: NOTE: You can supply different env vars to the following make commands these include:
+    * Version using the env var VERSION 
+    * Tag via the env var IMAGE_TAG for tags not following the semantic format.
+    * Image registry via the env var REGISTRY
+    * Registry org via the env var ORG
+
+    For example
+
+    `make bundle VERISON=2.0.1`
+    `make bundle IMAGE_TAG=asdf`
+    
+1. Generate the OLM bundle manifests for MGC, run the following make target:
+    ```
+    make bundle 
+    ```
+
+2. Build the bundle image
+
+    ```
+    make bundle-build
+    ```
+3. Push the bundle image to your image repository
+    ```
+    make bundle-push
+    ```
+
+### Creating the catalog
+
+1. Build the catalog image
+
+    ```
+    make catalog-build
+    ```
+2. Push the catalog image to your image repository 
+    ```
+    make catalog-push
+    ```   
+
+### Installing the operator via OLM catalog
+1. Create a namespace:
+```bash
+   cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: multi-cluster-gateways-system
+EOF
+```
+
+2. Create a catalog source:
+ ```bash
+    cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: mgc-catalog
+  namespace: olm
+spec:
+  sourceType: grpc
+  image: quay.io/kuadrant/multicluster-gateway-controller-catalog:latest
+  grpcPodConfig:
+    securityContextConfig: restricted
+  displayName: mgc-catalog
+  publisher: Red Hat
+EOF
+```
+
+3. Create a subscription
+```bash
+    cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: multicluster-gateway-controller
+  namespace: multi-cluster-gateways-system
+spec:
+  channel: alpha
+  name: multicluster-gateway-controller
+  source: mgc-catalog
+  sourceNamespace: olm
+  installPlanApproval: Automatic
+EOF
+```
+4. Create a operator group
+```bash
```

Comment:
is this something we have to create? I would have thought the sub was all you needed 🤷

Reply:
To install the controller in the namespace you want you need a operator-group otherwise it gets installed in the namespace operators